gunrock/app/hits/hits_app.cuh

```diff
@@ -249,11 +246,13 @@ double hits(
     data_graph.CsrT::row_offsets   .SetPointer((SizeT *)row_offsets, num_nodes + 1, gunrock::util::HOST);
     data_graph.CsrT::column_indices.SetPointer((VertexT *)col_indices, num_edges, gunrock::util::HOST);
```

Comment:
This can also be replaced by target if target is defined before the above two lines.

Reply:
I'll update this, thanks